apiserver/common/errors/errors.go

```diff
@@ -1,14 +1,15 @@
-// Copyright 2013 Canonical Ltd.
+// Copyright 2020 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package common
+package errors
```

Comment:
I think this should be in a package
apiserver/errors

rather than
apiserver/common/errors